src/renderer/pages/sshsetup/ConnectAccount.js

```diff
@@ -85,6 +102,31 @@ const ConnectAccount = observer(({ onNext, location }) => {
     trackEvent('setup-flow', `${sessionStore.provider}-connect`);
   }
 
+  function onCopyToClipboardClicked() {
+    const { url, state } = getOauthUrlsForBasicInfo(sessionStore.provider);
+
+    sessionStore.addState(state);
+    window.clipboard.writeText(url);
+    // Show toast
+    toaster.notify(
+      () => {
+        return (
+          <div className="py-2 px-4 rounded shadow-md bg-green-500 text-white font-medium text-center text-base mt-20 mr-4">
+            <h1 className="font-semibold text-left">Key copied to clipboard</h1>
+            <span className="mt-2">
+              Paste it in your browser to connect account
+            </span>
+          </div>
+        );
+      },
+      {
+        duration: 2500,
+        position: Position['top-right'],
+      }
+    );
+    trackEvent('setup-flow', `${sessionStore.provider}-connect`);
```

Comment:
Add respective analytics event to understand how many users use this instead of connect button?

Reply:
Done.